uvtools/tests/test_dspec.py

```diff
@@ -75,6 +75,55 @@ def test_delay_filter_2D(self):
         np.testing.assert_allclose(np.average(data,axis=1), np.average(dmdl,axis=1), atol=1e-3)
         np.testing.assert_allclose(np.average(dres,axis=1), 0, atol=1e-3)
     
+    def test_delay_filter_leastsq(self):
```

Comment:
It would be good to have explicit tests for `fourier_model`, and to break the tests for the 1d and 2d versions into different functions.

Reply:
I've done the first part, but kept the 1D and 2D tests together, as I don't think there's much to be gained by separating them.